docs/properties.md

```diff
@@ -128,7 +128,7 @@ namespace App\Livewire;
 
 use Livewire\Component;
 
-class ManageTodos extends Controller
+class ManageTodos extends Component
```

Comment:
This was wrong. I fixed at the same PR.

Reply:
@devajmeireles please split unrelated changes, like this one, into a new PR. Thank!

Reply:
> @devajmeireles please split unrelated changes, like this one, into a new PR. That way if Caleb doesn't merge this feature, then that docs typo will still be fixed. Thanks!

Make sense. Thank you! I'll update the PR.